playkit/src/main/java/com/kaltura/playkit/player/PKSubtitlePosition.java

```diff
@@ -110,10 +110,15 @@ public PKSubtitlePosition setPosition(int horizontalPositionPercentage, int vert
         float horizontalPosition = checkPositionPercentageLimit(horizontalPositionPercentage);
 
         if (horizontalAlignment == null) {
-            horizontalAlignment = Layout.Alignment.ALIGN_CENTER;
+            horizontalAlignment = Layout.Alignment.ALIGN_CENTER; // This is required because
```

Comment:
// This is required because. ???

Reply:
we can merge this code with the below I think
adding null check and set 
 else {
            this.horizontalPositionPercentage = DIMEN_UNSET;
             horizontalAlignment = Layout.Alignment.ALIGN_CENTER;
        }

Reply:
> // This is required because. ???

My Mistake.

> we can merge this code with the below I think
> adding null check and set
> else {
> this.horizontalPositionPercentage = DIMEN_UNSET;
> horizontalAlignment = Layout.Alignment.ALIGN_CENTER;
> }

Done. Now resetting the values using global variables.